examples/Wkt.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2017 United States Government as represented by the Administrator of the
+ * National Aeronautics and Space Administration. All Rights Reserved.
+ */
+
+requirejs(['../src/WorldWind',
+        './LayerManager'],
+    function (ww,
+              LayerManager) {
+        "use strict";
+
+        WorldWind.Logger.setLoggingLevel(WorldWind.Logger.LEVEL_WARNING);
+
+        var wwd = new WorldWind.WorldWindow("canvasOne");
+
+        var layers = [
+            {layer: new WorldWind.BMNGLayer(), enabled: true},
+            {layer: new WorldWind.CompassLayer(), enabled: true},
+            {layer: new WorldWind.CoordinatesDisplayLayer(wwd), enabled: true},
+            {layer: new WorldWind.ViewControlsLayer(wwd), enabled: true}
+        ];
+
+        for (var l = 0; l < layers.length; l++) {
+            layers[l].layer.enabled = layers[l].enabled;
+            wwd.addLayer(layers[l].layer);
+        }
+
+        // Example showing the usage of Well Known Text collection in real life.
+        var defaultLayer = new WorldWind.RenderableLayer("Wkt Shapes");
+        new WorldWind.Wkt("" +
+            "GEOMETRYCOLLECTION(" +
+            "   POLYGON ((40 -70, 45 -80, 40 -90)), " +
+            "   POINT (33 -75)," +
+            "   MULTIPOINT ((38 -70),(42 -75),(38 -80))," +
+            "   LINESTRING ((33 -75, 37 -80, 33 -85))," +
+            "   MULTILINESTRING ((38 -70, 42 -75, 38 -80),(43 -65, 47 -70, 43 -75))" +
+            ")").load(null, null, defaultLayer);
+        wwd.addLayer(defaultLayer);
+
+        // Using the callback mechanism presented in the Wkt parser to update the shapes as well as showing the information about the successful rendering.
+        var customCallbackLayer = new WorldWind.RenderableLayer("Wkt Shapes");
+        new WorldWind.Wkt("MULTIPOLYGON (((50 -60, 55 -70, 50 -80)),((30 -60, 35 -70, 30 -80)))").load(function completionCallback(wkt, objects){
```

Comment:
In this example, I think the callbacks should appear aligned vertically to convey their relationship.